### DISCUSSION TITLE
JSONSchemaType for a field that can be of type string or null

### DISCUSSION THREAD

**Discussion by nisargpatel-m** [json-schema-org/community#631] [Q&A]
For a given interface: 

```
interface Connection {
  kind: string;
  accountTable: string | null;
}

const connectionSchema: JSONSchemaType<Connection> = {
  type: 'object',
  properties: {
    kind: {
      type: 'string'
    },
    accountTable: {
      type: ["string", "null"]
    }
  },
  required: ['kind', 'accountTable'],
  additionalProperties: false,
};
```

the following JSONSchemaType validation is throwing an error. Playground link: https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgKQMoHkByqDGALAUxAEMAVATzALgF84AzKCEOAcmICsA3VgbgCh+wAHYwCUesRzUAwhGHCCOGMHmJ+cOAGsRAEwBccAM4woIgOYDNUnBACuo0sQBGAGwKGTZ4ebgAfOGE7V1cBGkFbYRM4SMVlVWFcQhJDNCwkojJKAgAeOQUlFXkAPjgAXnVNGGzDVghnDkLWABoNODAmKlhgAiNDBDbNHWEDSs1x6qparwtWQdpW8bgbe0cXd375qpq4AG0AIhmffea4faCQ-YBdefDNGkW4KAIARztgZ9Hd1mHdFrYVg4YE43ARWFdHsRdLpgEVhMRXAAFTriFS9QySVxGAitGgCCLyIwQdwAOlcEHMAApYoUEhkSABKIA

Reference used: 
- https://json-schema.org/understanding-json-schema/reference/type

**Comment by gregsdennis** [comment]
Hey there. Thanks for the question, but this might be something you need to take back to the maintainers of the library.

Your usage of JSON Schema looks fine to me, though.
